src/gui/tray/Window.qml

```diff
@@ -279,6 +318,13 @@ Window {
                                         color: parent.parent.hovered ? Style.lightHover : "transparent"
                                     }
                                 }
+
+                                Accessible.role: Accessible.MenuItem
+                                Accessible.name: text
+                                Accessible.description: qsTr("Quits the nextcloud client application")

```

Comment:
Same here, description likely not needed since "Exit Nextcloud" is explanatory. As long as icons and menus have explanatory text, the literal interface text usually doesn’t need long descriptions.

(This at least is how we handle it in the web interface.)